09_json/mohitnag/pkg/puppy/store/testdata/invalid-puppies.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "puppyid": 1,
+        "breed": "dog",
+        "colour": "white",
+        "value": "2"
+    },
+    {
+        "puppyid": 1
```

Comment:
I think this file should be called `duplicate-puppies.json` rather than invalid, because the second puppy has a value  of zero (the zero value) so it is not invalid - invalid is only < 0.

Reply:
Renamed